README.md

```diff
@@ -30,7 +30,6 @@
 <meta http-equiv="X-UA-Compatible" content="IE=edge">
 <meta name="viewport" content="width=device-width,initial-scale=1,minimum-scale=1,maximum-scale=1,user-scalable=no">
 <meta name="description" content="Description">
-<meta name="keywords" content="Keywords">
```

Comment:
Can you update the PR by moving this meta tag down from the must section?

Reply:
What would be an appropriate place to move it to?

Reply:
Can you move it down from the must tags comment?